composer.json

```diff
@@ -38,11 +38,13 @@
     },
     "suggest": {
         "ext-openssl": "To use Security::encrypt() or have secure CSRF token generation.",
-        "lib-ICU": "The intl PHP library, to use Text::transliterate() or Text::slug()"
+        "lib-ICU": "The intl PHP library, to use Text::transliterate() or Text::slug()",
+        "paragonie/csp-builder": "CSP builder, to use the CSP Middleware"
     },
     "require-dev": {
         "phpunit/phpunit": "^5.7|^6.0",
-        "cakephp/cakephp-codesniffer": "^3.0"
+        "cakephp/cakephp-codesniffer": "^3.0",
+        "paragonie/csp-builder": "^2.1"
```

Comment:
Won't this break builds on php5 going forwards? Could we conditionally include this dependency in travis instead of breaking the php5 builds?

Reply:
We can do that. Any idea how? I'm not really good with the scripts for Travis and don't want to produce 10 or 20 trial and error commits just to get it working.